docs/book/src/reference/metrics.md

```diff
@@ -73,6 +76,29 @@ for the metrics exported from the namespace where the project is running
+| workqueue_retries_total                       | Counter   | Total number of retries handled by workqueue.                                                                                                                                                                                        |
+| rest_client_requests_total                    | Counter   | Number of HTTP requests, partitioned by status code, method, and host.                                                                                                                                                               |
+| controller_runtime_reconcile_total            | Counter   | Total number of reconciliations per controller.                                                                                                                                                                                      |
+| controller_runtime_reconcile_errors_total     | Counter   | Total number of reconciliation errors per controller.                                                                                                                                                                                |
+| controller_runtime_reconcile_time_seconds     | Histogram | Length of time per reconciliation per controller.                                                                                                                                                                                    |
+| controller_runtime_max_concurrent_reconciles  | Gauge     | Maximum number of concurrent reconciles per controller.                                                                                                                                                                              |
+| controller_runtime_active_workers             | Gauge     | Number of currently used workers per controller.                                                                                                                                                                                     |
+| controller_runtime_webhook_latency_seconds    | Histogram | Histogram of the latency of processing admission requests.                                                                                                                                                                           |
+| controller_runtime_webhook_requests_total     | Counter   | Total number of admission requests by HTTP status code.                                                                                                                                                                              |
+| controller_runtime_webhook_requests_in_flight | Gauge     | Current number of admission requests being served.                                                                                                                                                                                   |
```

Comment:
these 3 are defined in: https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/webhook/internal/metrics/metrics.go#L28-L60

Reply:
Yeah, exactly.